docs/reference/commandline/stack_deploy.md

```diff
@@ -82,7 +82,7 @@ you can combine these by passing them to `docker-compose config` with the `-f` o
 and redirecting the merged output into a new file.
 
 ```bash
-$ docker-compose -f docker-compose.yml -f docker-compose.prod.yml config > docker-stack.yml
+$ docker-compose config -f docker-compose.yml -f docker-compose.prod.yml config > docker-stack.yml
```

Comment:
Actually, the `config` command is at the end here; the `-f` option is a global option (`docker-compose -f <foo> COMMAND`), so I think the original example is correct

Reply:
Oh sorry about that  !  I did not see the config at the end... Excuse me for the misunderstanding.

Reply:
No worries; I almost overlooked it as well!

Let me close this pull request, but thank you so much for contributing 👍